apim/values.yaml

```diff
@@ -363,19 +380,24 @@ gateway:
     maxReplicas: 6
     targetAverageUtilization: 50
     targetMemoryAverageUtilization: 80
+  # This is usefull in case of dynamic rewriting
+  entrypoint: /gateway/
```

Comment:
The same here.

Can you explain the purpose of this property ?
Do not see the purpose in the case of dyn rewriting, I'm probably missing something there...

Reply:
I could maybe use the description I put in the README `Defines the gateway entrypoint for the portal (In some case, this value is not needed because 'gateway.ingress.path' does not contain matching group and is usable as is for the gateway entrypoint)`
I'll work in the next few days on explaining this more clearly in the README, with some use case if needed